spec/lib/hosting/hetzner_apis_spec.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+RSpec.describe Hosting::HetznerApis do
```

Comment:
It's not for this PR, just future note. [webmock](https://github.com/bblimke/webmock) makes HTTP request stubbing easier.

Reply:
ah yeah we should definitely pull in webmock.

Reply:
done